Shell/Execution.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2020, The SereniyOS developers.
```

Comment:
Git is too darn smart for its own good...

Reply:
That typo tho ^^ You have the same one remaining in Shell.cpp / Shell.h fyi.

Reply:
lol I made a script that auto-adds these...and typo'd the darn thing.
good job me.

Reply:
Thanks for catching that.